src/silx/gui/dialog/GroupDialog.py

```diff
@@ -164,6 +164,26 @@ def getSelectedDataUrl(self):
         """
         return self._selectedUrl
 
+    def setSelectedUrl(self, url: DataUrl):
```

Comment:
Today the url `data_path` can be anything: a path to a dataset or a group.
On the `DatasetDialog` especially we can select a group and not necessarily a `h5py.Dataset`. But this is the original implement of the class. And to me this can make sense.